docs/contributing.md

```diff
@@ -64,6 +64,14 @@ We're writing documentation as we are developing new features. If you find
 something that is confusing or not covered at all, feel free to
 [open a bug][ob] or [contribute][cd]
 
+## Contributing to the Design
+
+Since Radicle is an open source project, anyone can contribute. This is normal in open source development, but we do it for design too! Here's how you can get started:
+
+I’d suggest starting by joining our [Discord server](ds) to chat with me or anyone on the team and ask any questions you have. It’s all public and open for anyone to join and chat in. We even have our “internal” chats public on there where we chat all day about features. The #Upstream channel is an example of that.
```

Comment:
Links have to be formatted like this, for them to render correctly: `[Discord server][ds]`.